tests/framework/integration/cluster.go

```diff
@@ -562,10 +562,11 @@ type Member struct {
 	Server        *etcdserver.EtcdServer
 	ServerClosers []func()
 
-	GrpcServerOpts []grpc.ServerOption
-	GrpcServer     *grpc.Server
-	GrpcURL        string
-	GrpcBridge     *bridge
+	GRPCServerOpts []grpc.ServerOption
+	GRPCServer     *grpc.Server
+	//revive:disable-next-line:var-naming
+	GrpcURL    string
```

Comment:
I'll address this change in a follow-up PR—the change yields about a hundred lines on changes.